src/os/bluestore/BlueStore.h

```diff
@@ -3836,7 +3837,7 @@ class BlueStoreRepairer
 	      uint64_t min_alloc_size,
 	      uint64_t mem_cap = DEF_MEM_CAP) {
       ceph_assert(!granularity); // not initialized yet
-      ceph_assert(min_alloc_size && isp2(min_alloc_size));
+      ceph_assert(min_alloc_size && std::has_single_bit(min_alloc_size));
```

Comment:
`std::has_single_bit` already takes care of 0 -- `isp2` should have been named `isp2_or_zero`:
```suggestion
      ceph_assert(std::has_single_bit(min_alloc_size));
```

Reply:
@idryomov fixed and repushed.